cernopendata_client/searcher.py

```diff
@@ -218,13 +219,12 @@ def get_file_info_remote(recid, filtered_files=None):
     convenience.
     :rtype: list
     """
-    server = "http://opendata.cern.ch"
     file_info_remote = []
-    record_json = get_record_as_json(server=server, recid=recid)
+    record_json = get_record_as_json(server=SERVER, recid=recid)
     for file_info in record_json["metadata"]["files"]:
         file_checksum = file_info["checksum"]
         file_size = file_info["size"]
-        file_uri = file_info["uri"].replace("root://eospublic.cern.ch/", server)
+        file_uri = file_info["uri"].replace("root://eospublic.cern.ch/", SERVER)
```

Comment:
This is another nice candidate for centralising in `config.py`, perhaps under the name EOSPUBLIC_ROOT_URI... Or we could have SERVER_HTTP_URI (currently named SERVER) and SERVER_ROOT_URI (for `root:/...`) for consistency.

Reply:
Yeah, this sounds good 👍

Reply:
what should be the `doc string` for `SERVER_ROOT_URI`?

Reply:
Something harmonious with the HTTP one, for example:

```python
SERVER_HTTP_URI = "http://opendata.cern.ch"
"""Default CERN Open Data server to query over HTTP protocol."""

SERVER_ROOT_URI = "root://eospublic.cern.ch"
"""Default CERN Open Data server to query over XRootD protocol."""
```

Note also trailing slash difference in the `replace` command, the XRootD protocol needs two slashes, so please beware to keep it there.

Reply:
Done 👍